### DISCUSSION TITLE
Feature request - Russian Language

### DISCUSSION THREAD

**Discussion by kaktoforum** [bcholmes/StarTrek2d20#191] [Ideas]
Good time of the day.
Will be glad to have a Russian translation within available languages.
Added it to Spreadsheet and started to translate but found request to have a request here.
So I'm requesting my request :D
Backed up already translated and holding additional inputs. Waiting for approval to continue.
If my changes will interrupt something, you can remove them.

**Comment by bcholmes** [comment]
Hi! I noticed your changes to the spreadsheet a little while ago. I'm totally up for including the Russian translations.

I think one area that's going to be a problem is that the font that I use for Headers and LCARS frame doesn't have great support for Cyrillic characters. Let me look in to that.

**Reply by kaktoforum** [reply]
I will check tomorrow which fonts I've used when "homebrewed" my GM screen.
Burned couple of hours to find something that fits.

**Reply by kaktoforum** [reply]
![photo_2024-02-14_18-05-18](https://github.com/bcholmes/StarTrek2d20/assets/77748199/ddae56d3-20a2-4f54-89e3-a75fe13f3461)
![photo_2024-02-14_18-05-11](https://github.com/bcholmes/StarTrek2d20/assets/77748199/63715435-0b93-4e26-81ab-d55b5c3227eb)

**Comment by kaktoforum** [comment]
@bcholmes 
I used "Impact" for Headers and "Verdana" for the text.
![image](https://github.com/bcholmes/StarTrek2d20/assets/77748199/5b7729a6-1d22-4ef6-bde1-92aaa6307d0c)


Original headers are "Star Trek BT"
EN Text is "Arial MT" for small black, "Verdana" for bullets , Cyrillic text is "72 Condenced"+ "Arial MT"

![image](https://github.com/bcholmes/StarTrek2d20/assets/77748199/41f1524e-e31c-42b6-8606-7e28195df407)
